crates/task-impls/src/vid.rs

```diff
@@ -361,6 +369,9 @@ where
                     }
                 }
             }
+            HotShotEvent::VidCertRecv(_) => {
```

Comment:
I actually think we'd want to put this event into the consensus task, since replicas need to check if they have it before voting on a quorum proposal.  But this can be done in a separate issue.